plotly/tools.py

```diff
@@ -3931,4 +3934,3 @@ def make_table_annotations(self):
                         font=dict(color=font_color),
                         showarrow=False))
         return annotations
-
```

Comment:
k, ye I just started using `atom`, and it doesn't seem to raise an issue about that anymore.